bundle/Resources/config/default_settings.yaml

```diff
@@ -0,0 +1,10 @@
+parameters:
+  netgen_ibexa_scheduled_visibility.enabled: false
```

Comment:
Four spaces should be here.

Reply:
fixed in c6fed8b415a0806f335cb67eaad3a1bce8cc5c7d